crates/ruma-events/src/room/message/audio.rs

```diff
@@ -13,7 +13,6 @@ use crate::room::{
 /// The payload for an audio message.
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
-#[serde(tag = "msgtype", rename = "m.audio")]
```

Comment:
Although I doubt anyone is using it, I feel like we must also document the change in serialization in the changelog, that the `msgtype` is no longer serialized for this types alone.

Reply:
Good idea. Have expanded the changelog entry.